leadconnectorhq.com.site-dynamic-record.json

```diff
@@ -12,7 +12,7 @@
   "records": [
     {
       "type": "A",
-      "host": "www",
+      "host": "%SUB1%",
```

Comment:
Why a variable here? You should be using "host" parameter for subdomains as it allows multiple instances and better handling of conflicts on DNS provider side.

![image](https://github.com/Domain-Connect/Templates/assets/28707030/1c7d25f8-355c-4108-b66a-c8fcd10daffe)

vs. 

![image](https://github.com/Domain-Connect/Templates/assets/28707030/fe963b23-7828-4c3d-a40b-551efc7ec0b7)